src/normalize/base.ts

```diff
@@ -43,5 +45,7 @@ export interface NormalizerParams {
   repeater?: RepeaterValue;
   repeaterPrefix?: string;
   selections?: TopLevelSelectionParameter[];
+  emptySelections?: Dict<boolean>;
```

Comment:
Should this be `Set` instead? Or can the value ever be `false`?

Reply:
It can be false, yes.